src/Phinx/Db/Adapter/AdapterInterface.php

```diff
@@ -347,6 +347,24 @@ public function hasTable($tableName);
      */
     public function createTable(Table $table, array $columns = [], array $indexes = []);
 
+    /**
+     * Changes the primary key of the specified database table.
+     *
+     * @param Table $table Table
+     * @param string|array|null $newColumns Column name(s) to belong to the primary key, or null to drop the key
+     * @return void
+     */
+    public function changePrimaryKey(Table $table, $newColumns);
+
+    /**
+     * Changes the comment of the specified database table.
+     *
+     * @param Table $table Table
+     * @param string|null $newComment New comment string, or null to drop the comment
+     * @return void
+     */
+    public function changeComment(Table $table, $newComment);
```

Comment:
This function and the one above should be moved to `DirectActionInterface`. The idea is to keep `AdapterInterface` small and all those direct actions should live in the tother interface: https://github.com/djcvijic/phinx/blob/74341755a952804e8698b18d0eee9c76c193a117/src/Phinx/Db/Adapter/DirectActionInterface.php

Reply:
Will do!

But I guess I'll do so still not understanding what the difference is between those interfaces. AdapterInterface seems to be more for higher-level operations and non-modifying operations in general, whereas DirectActionInterface seems to have operations that modify existing tables in some way. And the explanation in a comment on DirectActionInterface does not help much either.
`Represents an adapter that is capable of directly executing alter instructions, without having to plan them first.`

If future contributors need to understand this better, it might be worth to clarify the difference.

Reply:
To be honest `DirectActionInterface` are methods I don't want t all exposed, but they already existed and many depend on them, so I had to keep them.

The general idea is that all changes to the database should be planned, by crating `Action` classes. All methods in  `DirectActionInterface` directly modify the database, without going through a planning and validation step, so they cannot be optimised either.